clearpath_control/config/w200/control.yaml

```diff
@@ -0,0 +1,69 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 50  # Hz
+    use_sim_time: False
+
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    platform_velocity_controller:
+      type: diff_drive_controller/DiffDriveController
+
+platform_velocity_controller:
+  ros__parameters:
+    use_sim_time: False
+    left_wheel_names: [ "front_left_wheel_joint", "rear_left_wheel_joint" ]
+    right_wheel_names: [ "front_right_wheel_joint", "rear_right_wheel_joint" ]
+
+    wheel_separation: 1.08 
```

Comment:
These values should match https://github.com/warthog-cpr/warthog/blob/kinetic-devel/warthog_control/config/control.yaml#L21